core/src/kvs/tests/raw.rs

```diff
@@ -1,12 +1,12 @@
 use crate::key::error::KeyCategory::Unknown;
 
-#[tokio::test]
-#[serial]
-async fn initialise() {
-	let mut tx = new_tx(Write, Optimistic).await;
-	assert!(tx.put(Unknown, "test", "ok").await.is_ok());
-	tx.commit().await.unwrap();
-}
+// #[tokio::test]
```

Comment:
Why is this getting commented? Why did it exist initially 🤔 Maybe we can just delete it if it isnt doing anything. It is a test... looks like its just testing init.

Reply:
Oh my bad, just fixed it!